README.md

```diff
@@ -83,7 +83,22 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
 - Python 3.9+ (<3.12 for PyTorch JIT)
 - Approximately 60GB disk space (entire process)
 
-> **NOTE:** PyTorch 2.2.1 does not support `torch.compile` with Python 3.12. On Fedora 39+, install `python3.11-devel` and create the virtual env with `python3.11` if you wish to use PyTorch's JIT compiler.
+> **NOTE:** PyTorch 2.2.1 does not support `torch.compile` with Python 3.12. On Fedora 39+, install `python3.11-devel` and create the virtual env with `python3.11` if you wish to use PyTorch's JIT compiler. 
+
+> **NOTE:** When a version greater than Python 3.11 is already installed (e.g. on Mac) use pyenv to help manage python versions:
```

Comment:
Mac does not use PyTorch, so there is no issue with python 3.12 on Mac.

Reply:
Really? I couldn't get this working with the current instructions for 3.12. I must have missed a step or over did a step. Weird.

Reply:
Closed.